### ISSUE TITLE
[Component] [pagination] 使用分页组件，控制台会显示[Vue warn] Write operation failed: computed value is readonly

### ISSUE THREAD

**Issue by lycorisdeve** [element-plus/element-plus#16582]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.4.21`
- Element Plus Version: `2.7.0`
- Browser / OS: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36`
- Build Tool: `Vite`

### Related Component

- `el-pagination`

### Reproduction Link

[Docs CN](https://element-plus.gitee.io/zh-CN/component/pagination.html)

### Steps to reproduce

官网示例代码
```vue
<template>
  <div class="flex items-center mb-4">
    <el-radio-group v-model="small" class="mr-4">
      <el-radio-button :value="false">default</el-radio-button>
      <el-radio-button :value="true">small</el-radio-button>
    </el-radio-group>
    <div>
      background:
      <el-switch v-model="background" class="ml-2" />
    </div>
    <div class="ml-4">
      disabled: <el-switch v-model="disabled" class="ml-2" />
    </div>
  </div>

  <hr class="my-4" />

  <div class="demo-pagination-block">
    <div class="demonstration">Total item count</div>
    <el-pagination
      v-model:current-page="currentPage1"
      :page-size="100"
      :small="small"
      :disabled="disabled"
      :background="background"
      layout="total, prev, pager, next"
      :total="1000"
      @size-change="handleSizeChange"
      @current-change="handleCurrentChange"
    />
  </div>
  <div class="demo-pagination-block">
    <div class="demonstration">Change page size</div>
    <el-pagination
      v-model:current-page="currentPage2"
      v-model:page-size="pageSize2"
      :page-sizes="[100, 200, 300, 400]"
      :small="small"
      :disabled="disabled"
      :background="background"
      layout="sizes, prev, pager, next"
      :total="1000"
      @size-change="handleSizeChange"
      @current-change="handleCurrentChange"
    />
  </div>
  <div class="demo-pagination-block">
    <div class="demonstration">Jump to</div>
    <el-pagination
      v-model:current-page="currentPage3"
      v-model:page-size="pageSize3"
      :small="small"
      :disabled="disabled"
      :background="background"
      layout="prev, pager, next, jumper"
      :total="1000"
      @size-change="handleSizeChange"
      @current-change="handleCurrentChange"
    />
  </div>
  <div class="demo-pagination-block">
    <div class="demonstration">All combined</div>
    <el-pagination
      v-model:current-page="currentPage4"
      v-model:page-size="pageSize4"
      :page-sizes="[100, 200, 300, 400]"
      :small="small"
      :disabled="disabled"
      :background="background"
      layout="total, sizes, prev, pager, next, jumper"
      :total="400"
      @size-change="handleSizeChange"
      @current-change="handleCurrentChange"
    />
  </div>
</template>
<script lang="ts" setup>
import { ref } from 'vue'

const currentPage1 = ref(5)
const currentPage2 = ref(5)
const currentPage3 = ref(5)
const currentPage4 = ref(4)
const pageSize2 = ref(100)
const pageSize3 = ref(100)
const pageSize4 = ref(100)
const small = ref(false)
const background = ref(false)
const disabled = ref(false)

const handleSizeChange = (val: number) => {
  console.log(`${val} items per page`)
}
const handleCurrentChange = (val: number) => {
  console.log(`current page: ${val}`)
}
</script>

<style scoped>
.demo-pagination-block + .demo-pagination-block {
  margin-top: 10px;
}
.demo-pagination-block .demonstration {
  margin-bottom: 16px;
}
</style>

```

控制台：index.ts:17 [Vue warn] Write operation failed: computed value is readonly

### What is Expected?

No Warning

### What is actually happening?

 [Vue warn] Write operation failed: computed value is readonly

### Additional comments

_(empty)_

![image](https://github.com/element-plus/element-plus/assets/93499735/aba860eb-0bf8-4d6e-a092-3131ec4e6015)

**Comment by warmthsea** [comment]
我这边没有看到
![screenshots](https://github.com/element-plus/element-plus/assets/45450994/50acbfce-84e9-4876-8ad6-cf260daf6da0)

**Comment by warmthsea** [comment]
https://github.com/element-plus/element-plus/issues/16571

**Comment by btea** [comment]
Duplicate of #16571

**Comment by zhang-shengtao** [comment]
确实会显示，我使用的是2.7.0的版本， layout="total,sizes,prev,pager,next,jumper" 这个里面加sizes不会有这个，加了控制台就会有[Vue warn] Write operation failed: computed value is readonly 这个警告

**Comment by btea** [comment]
It has been fixed in 2.7.1, you can try it.
